tools/system_libs.py

```diff
@@ -1007,7 +1006,6 @@ class libcxx(NoBCLibrary, NoExceptLibrary, MTLibrary):
     'vector.cpp',
     os.path.join('experimental', 'memory_resource.cpp'),
     os.path.join('filesystem', 'directory_iterator.cpp'),
-    os.path.join('filesystem', 'int128_builtins.cpp'),
```

Comment:
why?

Reply:
its a duplicate of a file from compiler-rt.. atted to libc++ for gcc compat.. I added more info in the description.